src/checkers/inference/InferrableChecker.java

```diff
@@ -92,4 +92,9 @@ CFAnalysis createInferenceAnalysis(
      * @return true if should insert annotations of main modifier of local variables
      */
     boolean isInsertMainModOfLocalVar();
+
+    /**
```

Comment:
As discussed in person, add some javadocs, maybe use Unit Checker as an example, motivate why we need this method.